pom.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.0.M4</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>de.the-morpheus.e-du</groupId>
+  <artifactId>gateway-service</artifactId> <!-- D NOT CHANGE this! -->
+  <version>0.1-SNAPSHOT</version> <!-- D NOT CHANGE this! -->
+  <name>gateway-service</name> <!-- D NOT CHANGE this! -->
+  <description>Gateway-Microservice</description>
+
+  <properties>
+    <java.version>1.8</java.version>
```

Comment:
we do not have to set this since we are using a compiler plugin

Reply:
ok, we don't? Does this even compile without springs own plugin?

Reply:
otherwise we have to specifiy - somehow - the source and target level

Reply:
I added the maven compiler plugin